realm/realm-library/src/main/java/io/realm/internal/Table.java

```diff
@@ -43,11 +43,16 @@
     private static final long PRIMARY_KEY_FIELD_COLUMN_INDEX = 1;
     private static final long NO_PRIMARY_KEY = -2;
 
-    long nativePtr;
+    protected long nativePtr;
+    private static final long nativeFinalizerPtr;
     private final Context context;
     private final SharedRealm sharedRealm;
     private long cachedPrimaryKeyColumnIndex = NO_MATCH;
 
+    static {
```

Comment:
There might be a reason but I cannot remember ... maybe my brain was shorted at that time ... I will change it to setting directly.